src/commands/go-test-legacy.yaml

```diff
@@ -0,0 +1,48 @@
+steps:
+  - run:
+      name: Configuring Go Environment
+      command: |
+        mkdir -p /go/src/github.com/${CIRCLE_PROJECT_USERNAME}
+        ln -s /go/src/project /go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
+  - run:
+      name: Installing Dep
+      command: |
+        go get github.com/golang/dep/cmd/dep
+  - run:
+      name: Running Dep Ensure
+      command: |
+        export DEPPROJECTROOT=github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
+        dep ensure && git diff --exit-code
```

Comment:
I'd skip that part. I recon this will take ridiculous amount of time.

Reply:
CircleCI will terminate the job if it hits 10 minutes without output so at least it's not unbounded. I think there's value in ensuring that someone didn't forget to update their dependencies and vendor directory after modifying `Gopkg.toml` and that `vendor/` hasn't been manually edited.

Reply:
Ah this is ensure w/o -update. Yes this should be fine.